relay_client/src/lib.rs

```diff
@@ -104,6 +104,7 @@ impl ConnectionOptions {
         Ok(url)
     }
 
+    #[cfg(not(target_arch = "wasm32"))]
     fn as_ws_request(&self) -> Result<HttpRequest<()>, RequestBuildError> {
         use {
             crate::websocket::WebsocketClientError,
```

Comment:
`tokio_tungstenite::tungstenite::client::IntoClientRequest,`  is not available in wasm target